src/rqt_graph/rosgraph2_impl.py

```diff
@@ -0,0 +1,526 @@
+# Software License Agreement (BSD License)
```

Comment:
I don't see the reason to call this `rosgraph2_impl`. `rosgraph2` seems perfectly adequate and looks less clunky

Reply:
Also, for future reference, if you are copying in an entire file like this, it would be better to bring the original file in in one commit and then make changes to it in a second commit. Otherwise, your reviewer has to review the entire file

Reply:
I can't feasibly review this as it is. Please edit your git history so that the modifications are in a separate commit.